### PR TITLE
[Unformatted] Add mvc changes

### PR DESCRIPTION
**Pull request by Archit404Error**
## Overview
Unformatted version of #25 -- check it out to see all info about this PR. Approve #25 if this looks good

### CODE REVIEW

**Review by alanna-zhou — APPROVED** [review]
nice move to MVC! that's great :)

File: src/eatery/views.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+from django.http import JsonResponse
+from django.shortcuts import render
+from rest_framework.views import APIView
+
+from eatery.datatype.Eatery import EateryID
+
+from .controllers.update_eatery import UpdateEateryController
+
+
+class UpdateView(APIView):
```
Comment (alanna-zhou) [inline src/eatery/views.py:11]: i would rename this to something else
like be more descriptive bc you might have other views that are updating stuff as well that are contained in `eatery/views.py` later

**Review by alanna-zhou — COMMENTED** [review]
a nit but otherwise you can merge once someone else reviews this PR

@markim21 should review this before you merge! and she should write at least 1 question or comment about what is happening

File: src/eatery/views.py (hunk 1 above)
Comment (Archit404Error) [reply src/eatery/views.py:11]: Word

Comment (chalo2000) [reply src/eatery/views.py:11]: In fact, I'd recommend you name this EateryView or EaterysView based on if you're performing a CRUD operation for multiple items or a single item (take a look at how we do it on pear)

Comment (chalo2000) [reply src/eatery/views.py:11]: EaterysView would proably correspond to a route like `/api/eatery/` while EateryView would correspond to something like `/api/eatery/<id:int>`

**Review by chalo2000 — APPROVED** [review]
LGTM :) Before you rename UpdateView, take a look at my comments

**Review by markim21 — APPROVED** [review]
makes so much sense to separate things into the mvc format, this is a lot easier to read than before! I think next semester we should work on separating all the other models in api into their own groups so it looks like this

**Comment by Archit404Error** [comment]
Merged #25, so closing this PR.

**Pull request closed without merging**